modules/ingestor/src/service/mod.rs

```diff
@@ -166,15 +174,8 @@ impl IngestorService {
             .await
             .map_err(|err| Error::Storage(anyhow!("{err}")))?;
 
-        let stream = self
-            .storage
-            .retrieve(result.key())
-            .await
-            .map_err(Error::Storage)?
-            .ok_or_else(|| Error::Storage(anyhow!("file went missing during upload")))?;
-
```

Comment:
We were intentionally retrieving the doc after storing it to ensure that we could. If we don't find out now, we'll never know. Are we sure we want to eliminate this failsafe?

Reply:
The reason for retrieving it was actually because the stream was consumed at that point. From my side, it was never intended as a check.

Since none of the internal APIs makes actual use of that stream, it didn't seem reasonable to keep that pattern. We anyway load the full document in memory. So why store megabytes of SBOMs, and then load them again? If the don't trust S3 when it says "ok, I stored it" I think we have a different problem.